actiontext/app/models/action_text/rich_text.rb

```diff
@@ -39,10 +39,24 @@ class RichText < Record
     belongs_to :record, polymorphic: true, touch: true
     has_many_attached :embeds
 
+    after_initialize do
+      self.locale ||= I18n.locale
+    end
```

Comment:
Alternatively to this default, I wonder if it'd be best to extend the [has_rich_text](https://edgeapi.rubyonrails.org/classes/ActionText/Attribute.html#method-i-has_rich_text) class method to provide a means of determining the locale. Maybe something like:

```ruby
class Message < ApplicationRecord
  # this would be equivalent to `locale: -> { I18n.locale }`
  has_rich_text :content

  # accept a lambda
  has_rich_text :content, locale: ->(message) { message.some_other_column }

  # accepts a #to_proc
  has_rich_text :content, locale: :some_other_column
end
```

Would that be a good interface to include in addition to this `after_initialize` default?